ft/src/ft/kotlin/org/octopusden/octopus/vcsfacade/VcsFacadeGiteaTest.kt

```diff
@@ -7,7 +7,7 @@ import java.util.stream.Stream
 class VcsFacadeGiteaTest :
     BaseVcsFacadeFuncTest(
         GiteaTestClient("http://localhost:3000", GITEA_USER, GITEA_PASSWORD),
-        "git@gitea:3000:%s/%s.git"
+        "ssh://git@gitea:3000:%s/%s.git"
```

Comment:
```suggestion
        "ssh://git@gitea:3000/%s/%s.git"
```
правда придется поправить конфигурацию Gitea

Reply:
@aryabokon конфигурацию править не придётся. Тут ситуация печальнее, данный параметр используется как шаблон для генерации url под тестовые клиенты. 

А в GiteaTestClient есть метод, которые ожидает именно двоеточие и ничто иное:
`override fun convertSshToHttp(vcsUrl: String): String = "$url/${vcsUrl.substring(vcsUrl.lastIndexOf(":") + 1)}"`

Протестировать работоспособность url'ов со слешем вместо двоеточия не получится без исправления тестового клиента или перелопачивания архитектуры тестов.

Я поставлю задачу на улучшение тестовых клиентов. По хорошему, в этом методе еще один потенциальный баг - мы вообще не сверяем что в оригинальном ssh url (vcsUrl) был точно такой же хост, что и в http url ($url).

Reply:
@aryabokon с методом parseUrl в принципе та же история. Кажется в тестовых клиентах нужно работать по аналогичным правилам парсинга url'ов, как и в сервисах VCS Facade.

Бонусная проблема - нельзя использовать vcsUrl (даже приведенный к lowercase) как идентификатор репозитория. Нужно использовать распознанную пару project/organization/namespace + repository.

Reply:
@aryabokon тестовые клиенты добработаны и подключены.